src/schema/v2/vanityURLEntity.ts

```diff
@@ -34,13 +34,19 @@ const VanityURLEntity: GraphQLFieldConfig<void, ResolverContext> = {
   },
   resolve: (_root, { id }, { profileLoader, partnerLoader, fairLoader }) =>
     profileLoader(id).then((profile) => {
-      const ownerType = profile.owner?.type ?? profile.owner_type // what is the difference between these fields?
-      switch (ownerType) {
+      switch (profile.owner_type) {
         case "Gallery":
-        case "PartnerGallery":
+        case "Partner":
         case "Institution":
```

Comment:
i think we don't need `"Gallery"` or `"Institution"` now since `owner_type` doesn't remove the `Partner*` prefix from the type name.

Reply:
👍 Yeah I don't think we need `Partner` either, I'll find examples of each of these types when doing end to end testing on this.